dataflux_pytorch/benchmark/checkpointing/singlenode/README.md

```diff
@@ -106,7 +103,7 @@ Dataflux's implementation of CheckpointIO for PyTorch Lightning is undergoing ac
    </td>
   </tr>
   <tr>
-   <td style="background-color: #f3f3f3">Dataflux
+   <td style="background-color: #f3f3f3">Dataflux (non-multipart)
```

Comment:
Since multipart is the default and we don't have any scenarios where we recommend disabling multipart (right?) let's just present the multipart numbers as "Dataflux" and remove the non-multipart numbers. What do you think? We don't want to overwhelm users with information and make it harder to understand what performance to expect out of the box

Reply:
sounds good to me, will do